os/drivers/lcd/lcd_logo.h

```diff
@@ -25,6 +25,9 @@
 #ifdef CONFIG_LCD_LOGO_52_340
 #define LOGO_XRES 52
 #define LOGO_YRES 340
+#else
+#define LOGO_XRES 52
+#define LOGO_YRES 340

```

Comment:
The same values look strange. if we set them with all 0 in this conditional, what happen?

Reply:
if logo_xre and logo_yres is set to 0
then, it will bydefault show black screen
because it won't go inside this [while loop](https://github.com/Samsung/TizenRT/pull/6543/files#diff-0dcbbbfee32bf5989cf8e2ad4117f850d4a44a336017e1f0e7f7e4ef484d5e2fL439:~:text=*%202))%20%7B-,while%20(logo_arr_index%20%3C%20(LOGO_YRES%20*%20LOGO_XRES%20*%202))%20%7B,lcd_logo_fullscreen_data%5Blcd_data_index%5D%20%3D%20lcd_logo_raw_data%5Blogo_arr_index%2B%2B%5D%3B,-lcd_init_fullscreen_image%5Blcd_data_index)

I have changed it to 0.

Reply:
so, by default it will be black screen if no logo is selected